gulpfile.js

```diff
@@ -1,11 +1,12 @@
+require("dotenv").config()
 const gulp = require("gulp")
 const sass = require("gulp-sass")
 const autoprefixer = require("gulp-autoprefixer")
 const sourcemaps = require("gulp-sourcemaps")
 const uglify = require("gulp-uglify")
 const concat = require("gulp-concat")
 const browserSync = require('browser-sync').create()
-const port = 2999
+const port = process.env.PORT

```

Comment:
We can change this to: `const port = process.env.PORT || 3000`
Setting up via the .env variables but also providing a back-up just in case.

Reply:
I'm with Daniel on this.
I also use the || when it comes to the port, plus I like 3000 more than 2999